utils/s2n_safety.c

```diff
@@ -21,8 +21,17 @@
 #include <sys/types.h>
 #include <stdint.h>
 
+#include "utils/s2n_safety.h"
 #include "s2n_annotations.h"
 
+int S2N_TLS13_ENABLED = 0;
```

Comment:
nit: Variable names are typically lower cased.

Personally, I prefer wrapping this in a naive getter `s2n_get_tls13_support/s2n_is_tls13_supported()` unless someone wants to make a performance argument related to function call verhead(the compiler should inline this).